src/test/cc/math/distributed_discrete_gaussian_noiser_test.cc

```diff
@@ -83,15 +93,67 @@ TEST(DiscreteGaussianNoiserIndividualComponent, MinMaxShouldBeCorrect) {
   ASSERT_GE(min_value, 0);
 }
 
-TEST(DiscreteGaussianNoiserGlobalSummation,
-     ProbabilityMassFunctionShouldBeCorrect) {
+TEST(DiscreteGaussianNoiserGenerateNoiseComponentSamples,
+     ProbabilityMassFunctionShouldBeCorrectWithOneContributor) {
   int64_t contributor_count = 1;  // 1 contributor
+  double sigma_distributed = 48.231;
+  int64_t offset = 261;
+
+  DistributedDiscreteGaussianNoiseComponentOptions options = {
+      .contributor_count = contributor_count,
+      .sigma_distributed = sigma_distributed,
+      .truncate_threshold = offset,
+      .shift_offset = offset};
+  DistributedDiscreteGaussianNoiser distributed_discrete_gaussian_noiser(
+      options);
+
+  int64_t min_output = 0;
+  int64_t max_output = 2 * offset;
+
+  size_t num_trials = 10000;
+  std::map<int64_t, size_t> frequency_distribution;
+
+  for (size_t i = 0; i < num_trials; ++i) {
+    ASSERT_OK_AND_ASSIGN(
+        int64_t temp,
+        distributed_discrete_gaussian_noiser.GenerateNoiseComponent());
+    ASSERT_GE(temp, min_output);
+    ASSERT_LE(temp, max_output);
+    ++frequency_distribution[temp];
+  }
+
+  std::map<int64_t, double> expected_probability_distribution;
```

Comment:
Maybe separate the computation of `expected_probability_distribution` out so that we do not have to repeat this many times.